lib/types.ts

```diff
@@ -73,6 +75,7 @@ export interface Clue {
   number: number
   answer: string
   position: Position
+  splits?: number[]
```

Comment:
Should splits be required? The way I wrote the code, they will always be present in JSON because the code is just simpler that way, but they won't be required when converting from JSON back to `.xd`.

Reply:
No, I think optional is the right answer here 👍🏻